### PR TITLE
Update apache_echarts.json

### PR DESCRIPTION
**Pull request by pissang**
Added language faceting

Thanks for the help.

### CODE REVIEW

**Comment by shortcuts** [comment]
Hey, if you'd our scraper to retrieve `language` you need to add [this selector](https://github.com/algolia/docsearch-configs/blob/master/configs/vuepress.json#L33-L37) if available, or directly [from the URLs](https://docsearch.algolia.com/docs/config-file/#using-regular-expressions)

**Comment by pissang** [comment]
Hi, I update it to the "from the URLs" method you suggested.  Could you please help check if it's correct? Thanks a lot!

File: configs/apache_echarts.json [hunk 1]
```diff
@@ -13,9 +18,6 @@
     "lvl5": ".nuxt-content h6",
     "text": ".nuxt-content p, .nuxt-content li"
   },
-  "custom_settings": {
-    "attributesForFaceting": ["language"]
-  },
```
Comment (shortcuts) [inline configs/apache_echarts.json]: This can be restored with `lang` instead of `language` and it's good!

Comment (pissang) [reply configs/apache_echarts.json]: Fixed! Thanks!

**Pull request merged**